test/ui/widgets/post_tile/post_tile_view_model_test.dart

```diff
@@ -44,17 +44,17 @@ void main() {
         // arrange
         postTileViewModel.init(mockPost1);
         when(mockNavigationService.pushNamed(
-          ViewRoutes.post_details,
-          arguments: mockPost1,
+          Routes.postDetailsView,
+          arguments: PostDetailsViewArguments(post: mockPost1),
         )).thenAnswer((realInvocation) async => Null);
 
         // act
         postTileViewModel.showPostDetails();
 
         // assert
         verify(mockNavigationService.pushNamed(
-          ViewRoutes.post_details,
-          arguments: mockPost1,
+          Routes.postDetailsView,
+          arguments: isA<PostDetailsViewArguments>(),
```

Comment:
Shouldn't this line be 
```dart
PostDetailsViewArguments(post: mockPost1)
```
?

Reply:
That doesn't work, apparently isA is the right thing to do here...